SonarQube.Common/AnalysisConfig/AnalysisConfig.cs

```diff
@@ -35,11 +35,24 @@ public class AnalysisConfig
         public string SonarProjectVersion { get; set; }
 
         public string SonarProjectName { get; set; }
-        
+
+        /// <summary>
+        /// List of additional configuration-related settings
+        /// e.g. the build system identifier, if appropriate.
+        /// </summary>
+        /// <remarks>These settings will not be supplied to the sonar-runner.</remarks>
+        public List<ConfigSetting> AdditionalConfig { get; set; }
+
+        /// <summary>
+        /// List of analysis settings inherited from the SonarQube server
+        /// </summary>
+        public AnalysisProperties ServerSettings{ get; set; }
+
         /// <summary>
-        /// List of additional analysis settings
+        /// List of analysis settings supplied locally (either on the
+        /// command line or in a file)
         /// </summary>
-        public List<AnalysisSetting> AdditionalSettings { get; set; }
```

Comment:
Previously config settings, server properties and analysis properties were all stored in this single list.
They are now stored in separate properties in the config file.